lib/puppet/modulebuilder/builder.rb

```diff
@@ -5,15 +5,29 @@
 module Puppet::Modulebuilder
   # Class to build Puppet Modules from source
   class Builder
-    DEFAULT_IGNORED = [
+    # Due to the way how PathSpec generates the regular expression,
+    # `/*` doesn't match directories starting with a dot,
+    # so we need `/.*` as well.
+    IGNORED = [
+      '/*',
       '/.*',
-      '/pkg/',
-      '~*',
-      '/coverage',
-      '/checksums.json',
-      '/REVISION',
-      '/spec/fixtures/modules/',
-      '/vendor/'
+      '!/manifests',
+      '!/README*',
+      '!/metadata.json',
+      '!/LICENSE',
+      '!/hiera.yaml',
+      '!/data',
+      '!/templates',
+      '!/files',
+      '!/CHANGELOG*',
+      '!/docs',
+      '!/REFERENCE.md',
+      '!/locales',
+      '!/scripts',
+      '!/tasks',
+      '!/plans',
+      '!/types',
+      '!/bolt_plugin.json',
```

Comment:
may add .github here as well?

Reply:
`.github` is currently not allowed within a module release (https://github.com/puppetlabs/puppet-specifications/pull/157). What kind of files do you see in that directory that should be released?